test/functional/Java11andUp/src/org/openj9/test/condy/GarbageCollectionCondyTest.java

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package org.openj9.test.condy;
+
+import org.openj9.test.util.CircularBuffer;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.log4testng.Logger;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import javax.management.RuntimeErrorException;
+
+public class GarbageCollectionCondyTest {
+	private static final int CLASS_UNLOADING_ITERATIONS = 100000;
+	private static final int CLASSESS_KEPT_ALIVE_AT_ANY_TIME = 100;
```

Comment:
You have to run this with `-Xverbose:gc` and/or jconsole to figure out what the correct value is for `CLASSESS_KEPT_ALIVE_AT_ANY_TIME` and -Xmx/Xms.

Reply:
In this case `CLASS_UNLOADING_ITERATIONS` is misleading, because we are not trying to unload classes, but we want to create enough garbage such that the GC walks the constant pools